pkgs/applications/misc/xpdf/cmake_version.patch

```diff
@@ -8,7 +8,7 @@ https://cmake.org/cmake/help/v3.10/command/cmake_minimum_required.html
  #
  #========================================================================
  
--cmake_minimum_required(VERSION 2.8.8)
+-cmake_minimum_required(VERSION 2.8.12)
```

Comment:
This is too short to deserve a file. What about `substituteInPlace`?

Reply:
fixed

Reply:
>This is too short to deserve a file. What about substituteInPlace?

Generally I prefer patches, because if they fail to apply you'll know. But in this case I guess a silently failing substituteInPlace will still lead to a noticeable build failure.